test/JDBC/expected/ISC-Tables-vu-verify.out

```diff
@@ -1,14 +1,33 @@
-SELECT * FROM information_schema.tables WHERE TABLE_NAME = 'isc_tables_vu_prepare_t1'
-SELECT * FROM information_schema.tables WHERE TABLE_NAME = 'ISC_TABLES_VU_PREPARE_T1'
+-- Rename the existing tables to fix the originally stored incorrect bbf_original_rel_name
```

Comment:
I'd say this rename and fix should not be at line 1, since I'd like to also get the impression that normal mVU/MVU won't fix this, so pls put the rename step after verify the wrong result, and then rename, and then show the correct result.